### PR TITLE
Adding params to VerifyCredentials to allow gettign emails from users if possible

### PR DESCRIPTION
**Pull request by jquacinella**
This is in reference to: https://github.com/bear/python-twitter/issues/235

---
This change is [Reviewable](https://reviewable.io/reviews/bear/python-twitter/376)

### CODE REVIEW

File: twitter/api.py [hunk 1]
```diff
@@ -4556,16 +4556,35 @@ def GetUserStream(self,
...
+        data = {}
+        if include_entities:
+            data['include_entities'] = 1
+        if skip_status:
+            data['skip_status'] = 1
+        if include_email:
+            # TODO: not sure why but the twitter API needs string true, not a 1
+            data['include_email'] = 'true'
+        resp = self._RequestUrl(url, 'GET', data)  # No_cache
+        data = self._ParseAndCheckTwitter(resp.content)
```
Comment (jeremylow) [inline twitter/api.py]: need to decode the resp for python3. should be

``data = self._ParseAndCheckTwitter(resp.content.decode('utf-8'))``

**Comment by jeremylow** [comment]
Looks good to me. Do you know what happens when the app isn't white listed and email is requested?

**Comment by jquacinella** [comment]
Good question. I put in the original comment "When set to true email will be returned in the user objects as a string. If the user does not have an email address on their account, or if the email address is un-verified, null will be returned". However, if the app is not white-listed, I am not sure. My assumption would be that it'll be null as well, but I would have to set this up as a use case and see. I would only be able to do that later this week / weekend.

**Comment by jeremylow** [comment]
I don't have a whitelisted app, so I don't know what the actual data that comes back looks like when `include_email` is set to `True`, but when your app isn't whitelisted, there's no error thrown; the `email` key doesn't exist in the json response.

I think this is good to merge.

**Pull request merged**